src/regdesk/printer_manager.js

```diff
@@ -184,6 +185,36 @@ export class PrinterManager {
     await this.#minorDropdown.printer.printLabel(label);
   }
 
+  /**
+   * Print a label buider to the printer determined by the label builder.
+   * @param {BadgeLabelBuilder} builder
+   */
+  async printLabelBuilder(builder) {
+    var label;
+    switch (builder.isMinor) {
+      case true:
```

Comment:
The mental pattern I jumped to here is that isMinor is "a badge state modifier" and, in my experience, these types of modifiers grow. I then automatically jumped to "okay well if there's going to be more in the future this is deserving of a switch/case to handle them all". [C# has some interesting semantics around the switch/case keyword](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/switch-expression#case-guards) that makes it much more common instead of chains of if statements, which is why I reached for it first.

That said, it's unlikely this specific iteration of the code will survive very long for that to be valuable. I'll swap it around to ifs.